init.lua

```diff
@@ -1,95 +1,142 @@
+vim.opt.clipboard = "unnamedplus"
 vim.opt.modifiable = true -- buffers per default modifiable
 vim.opt.encoding = "UTF-8"
 
-
-
 --
--- Key Bindings 
+-- NeoVim Keymappings
 --
 
--- File browser 'netrw'
-vim.keymap.set("n", "<leader>e", vim.cmd.Ex)
+-- Open file browser netrw
+-- vim.keymap.set("n", "<leader>e", vim.cmd.Ex)
 
--- Move selected lines
+-- Move highlighted parts
 vim.keymap.set("v", "K", ":m '<-2<CR>gv=gv") -- move up
 vim.keymap.set("v", "J", ":m '>+1<CR>gv=gv") -- move down
 
 -- Remaps
-vim.keymap.set("i", "<C-c>", "<ESC>") -- CTRL-C -> ESC
+vim.keymap.set("i", "<C-c>", "<ESC>")
+
+
+vim.keymap.set("i", "<C-BS>", "<C-W>")
+
+vim.keymap.set("n", "<C-1>", "<C-w>o")
+vim.keymap.set("n", "<C-2>", "<C-w>v")
+vim.keymap.set("n", "<M-C-2>", "<C-w>v<C-w><C-w>")
+vim.keymap.set("n", "<C-3>", "<C-w>s")
+
+vim.keymap.set("n", "<C-.>", "<cmd>lua vim.lsp.buf.code_action()<CR>")
+vim.keymap.set("v", "<C-.>", "<cmd>CodeActionMenu<CR>")
+
+vim.keymap.set("n", "<M-C-9>", "<cmd>Telescope git_status theme=ivy<CR>")
+vim.keymap.set("n", "<C-0>", "<cmd>Telescope projects<CR>")
+
+
+vim.keymap.set("n", "<M-C-S-F>", ":lua require'fzf-lua'.live_grep({ cwd=\"~/Software\" })<CR>")
+vim.keymap.set("n", "<M-F>", "<cmd>FzfLua files cwd=~/<CR>")
+vim.keymap.set("n", "<M-f>", "<cmd>FzfLua files cwd=~/Software/<CR>")
+vim.keymap.set("n", "<M-C-F>", "<cmd>FzfLua blines<CR>")
+vim.keymap.set("n", "<C-4>", "<cmd>FzfLua live_grep_native<CR>")
+vim.keymap.set("n", "<C-7>", "<cmd>FzfLua oldfiles<CR>")
+vim.keymap.set("n", "<C-8>", "<cmd>FzfLua buffers<CR>")
+vim.keymap.set("n", "<C-9>", "<cmd>FzfLua git_files<CR>")
+vim.keymap.set("n", "<C-ß>", "<cmd>FzfLua lsp_live_workspace_symbols<CR>")
+vim.keymap.set("n", "<M-o>", "<cmd>FzfLua jumplist<CR>")
+
+vim.keymap.set("n", "<C-f>", "/")
 
+vim.keymap.set("n", "<A-j>", "}")
+vim.keymap.set("n", "<A-k>", "{")
+vim.keymap.set("v", "<A-j>", "}")
+vim.keymap.set("v", "<A-k>", "{")
 
+vim.keymap.set("n", "<C-j>", "<cmd>lua vim.diagnostic.goto_next()<CR>")
+vim.keymap.set("n", "<C-k>", "<cmd>lua vim.diagnostic.goto_prev()<CR>")
+
+vim.keymap.set("n", "<M-NL>", ":cnext<CR>")
+vim.keymap.set("n", "<M-C-K>", ":cprev<CR>")
+
+vim.keymap.set("n", "<M-C-O>", "g;")
+vim.keymap.set("n", "<M-Tab>", "g,")
+
+vim.keymap.set("n", "<A-S-j>", "<cmd>lua require 'gitsigns'.next_hunk()<cr>")
+vim.keymap.set("n", "<A-S-k>", "<cmd>lua require 'gitsigns'.prev_hunk()<cr>")
+
+vim.keymap.set("n", "<A-S-o>", "<cmd>TypescriptOrganizeImports<CR>")
+vim.keymap.set("n", "<A-S-i>", "<cmd>TypescriptAddMissingImports<CR>")
+vim.keymap.set("n", "<A-S-r>", "<cmd>TypescriptRemoveUnused<CR>")
+
+vim.keymap.set("i", "<C-s>", "<ESC>")
+vim.keymap.set("n", "<C-s>", ":w<CR>")
+
+vim.keymap.set("n", "<C-tab>", ":e#<CR>")
+
+vim.keymap.set("i", "<C-Enter>", "<C-o>O")
+
+vim.keymap.set("n", "<A-u>", "<cmd>noh<cr>")
 
--- 
 --
 -- Windows specific configuration
--- ./lua/windows.lua
 --
 
 if vim.fn.has('win32') then
-  pcall(require, 'windows')
```

Comment:
The motivation for pcall was, that Neovim is also working without the windows.lua file. You will get an error with "require".

Reply:
Whoops cant even remember that I changed it. I blame the kids!